packages/app/src/server/models/bookmark-folder.ts

```diff
@@ -26,12 +31,20 @@ export interface BookmarkFolderModel extends Model<BookmarkFolderDocument>{
   findChildFolderById(parentBookmarkFolder: Types.ObjectId | string): Promise<BookmarkFolderDocument[]>
   deleteFolderAndChildren(bookmarkFolderId: Types.ObjectId | string): {deletedCount: number}
   updateBookmarkFolder(bookmarkFolderId: string, name: string, parent: string): BookmarkFolderDocument | null
+  insertOrUpdateBookmarkedPage(pageId: IPageHasId, userId: Types.ObjectId | string, folderId: string)
```

Comment:
Is not a return type and promise required?
`insertOrUpdateBookmarkedPage(pageId: IPageHasId, userId: Types.ObjectId | string, folderId: string): Promise<BookmarkFolderDocument>`

Same question for other `BookmarkFolderModel`'s return types.